RELEASE-CHECKLIST.md

```diff
@@ -1,6 +1,7 @@
 # Release Checklist for the SPDX Java Tools
 
 - [ ] Check for any warnings from the compiler and findbugs
+- [ ] Run unit tests for all packages that depend on the library
```

Comment:
I wonder if we should mention the "slow tests" here too?

e.g.
- [ ] Run unit tests with `export SPDX_JAVA_LIB_RUN_SLOW_TESTS=true` and ensure extended (slow) test suite passes

Reply:
Good idea - this probably would have caught the issue in the last release.